### ISSUE TITLE
513*513train  need 22GB memory ?

### ISSUE THREAD

**Issue by teleger** [chengyangfu/caffe#12]
i  need.   low   memory   ..   so  ... config file. set. or add   pruneing file  of model.. thanks..  from Chinese.

**Comment by chengyangfu** [comment]
Yes, that is a problem I faced during training. 
I am really lucky to access some m40/p40 machines which contain 24 GB. 
I would suggest to use smaller batch size, freeze the batch normalization layers in Resnet, and don't use the batch normalization layers in the newly added layers.
